src/main/java/com/rakovets/course/java/core/practice/decision_making_statements/Task13.java

```diff
@@ -32,8 +32,22 @@ public static void main(String[] args) {
      * @return номер этажа
      */
     static int getFloorNumber(int numberFloors, int numberApartmentsPerFloor, int apartmentNumber) {
-        //TODO
-        // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0;
+        int amountOfApartments = 100;
+        int amountOfPorchs = 4;
+        int amountOfApartsInThePorch;
+        int floor;
+
+        amountOfApartsInThePorch = amountOfApartments / amountOfPorchs;
+
+        if (apartmentNumber % amountOfApartsInThePorch == 0){
+            floor = numberFloors;
+        }
+        else {
+            floor = (apartmentNumber % amountOfApartsInThePorch) / numberApartmentsPerFloor;
+            if (apartmentNumber % numberApartmentsPerFloor != 0){
+                floor++;
+            }
+        }
+        return floor;
```

Comment:
"Violation: There isn't separating white spaces (OR line break) for curly brake
Solution: Add white spaces"